themes/rad/layouts/partials/index-footer.html

```diff
@@ -2,13 +2,10 @@
   <div class="footer-content">
     <ul>
       {{ range .Site.Menus.footer }}
-      <li><a href="{{ .URL | relLangURL }}">{{ .Pre }}{{ .Name }}</a></li>
+      <li><a href="{{ .URL | safeURL }}">{{ .Pre }}{{ .Name }}</a></li>
```

Comment:
@xla is this gonna break something. We need to do safeUrl otherwise the `irc://` and `mailto:` stuff doesn't work